pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -1075,6 +1096,21 @@ func (asw *actualStateOfWorld) GetMountedVolumesForPod(
 	return mountedVolume
 }
 
+func (asw *actualStateOfWorld) GetMountedVolumeForPodByOuterVolumeSpecName(
+	podName volumetypes.UniquePodName, outerVolumeSpecName string) (MountedVolume, bool) {
```

Comment:
No caller uses `MountedVolume` return value. It should just return bool and be called ... `VolumeMountedWithOuterName` (following `VolumeExistsWithSpecName`)?

Reply:
The return value is used in the follow-up commit https://github.com/Lucaber/kubernetes/commit/ba1a3eb1f5719226c9d12a3f563f0ac58f60e3f0 . I don't think an additional function without the return value makes a lot of difference, this allocation is tiny compared to the previous one. Most of the cpu time is now used to iterate over the huge `attachedVolumes` map.